### DISCUSSION TITLE
Git for Windows v2.46.2.windows.1

### DISCUSSION THREAD

**Discussion by gitforwindowshelper** [git-for-windows/git#5170] [Announcements]
Changes since Git for Windows v2.46.1 (September 18th 2024)

### New Features

* Comes with [Git v2.46.2](https://github.com/git/git/blob/v2.46.2/Documentation/RelNotes/2.46.2.txt).
* Comes with [OpenSSH v9.9.P1](https://github.com/openssh/openssh-portable/releases/tag/V_9_9_P1).
* Comes with [MinTTY v3.7.6](https://github.com/mintty/mintty/releases/tag/3.7.6).
* Comes with [Bash v5.2.37](https://git.savannah.gnu.org/cgit/bash.git/commit/?id=6794b5478f660256a1023712b5fc169196ed0a22).
* Comes with the [new, experimental `--full-name-hash` option for `git repack`](https://github.com/git-for-windows/git/pull/5157) that helps packing monorepos more tightly.

Filename | SHA-256
-------- | -------
Git-2.46.2-64-bit.exe | eac009616605ec7207fbe1990627f453b826a1f23a33d54d9b0be8f4b0cb2094
Git-2.46.2-32-bit.exe | 6ca9019abb3aa963d81414d46fc054707efdf36d68b946ffdde6af8a6a374e46
PortableGit-2.46.2-64-bit.7z.exe | 58ae5c1a27d3f9ce8518c1a187860973cfa4dd604f518350e84f7baf418fc9fa
PortableGit-2.46.2-32-bit.7z.exe | 7351e28f498010f0aa2fd63abbd704698db38cf6f881e2417fb3d73e5a7e6f87
MinGit-2.46.2-64-bit.zip | 0dca60869825ceb8b6108be69f0c536174fbca45e11300f2c14c34632d8238ed
MinGit-2.46.2-32-bit.zip | 21a85dde40b3cb162ca6d0f6eda4007c3fac7243e2c63c34de979fc8cc4b9134
MinGit-2.46.2-busybox-64-bit.zip | a5c3ef14d4011324fc8b66e6e8943d8470b6d0606778be4d1f73f03ff9973c71
MinGit-2.46.2-busybox-32-bit.zip | d12e006af9c4d4f520a86a6c3cf03a1426433a19281574fc8f7b6d9d49f636ab
Git-2.46.2-64-bit.tar.bz2 | f05cf325a8bfaad6da2411e03065b5a4f2a1a69b9d1f9a258db1404524613610
Git-2.46.2-32-bit.tar.bz2 | 6fe9c7faf1e088b4be37945fa81bbc5252a8a7cb38c617925d5cc6c269cbc3dd

<hr /><em>This discussion was created from the release <a href='https://github.com/git-for-windows/git/releases/tag/v2.46.2.windows.1'>Git for Windows v2.46.2.windows.1</a>.</em>

**Comment by GeorgeSandcroft** [comment]
I would like to download and test OpenSSH v9.9 on a Windows Server
